app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/newConversationModal.tsx

```
@@ -1,5 +1,6 @@
 import { useRouter } from "next/navigation";
 import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { X } from "lucide-react";
```

ESLint

[error] Delete `react";⏎import·{·X·}·from·"lucide-`

(prettier/prettier)